python/sanic/server.py

```diff
@@ -17,3 +21,11 @@ async def user_info(request, id):
 @app.route("/user", methods=["POST"])
 async def user(request):
     return text("")
+
+if __name__ == "__main__":
+    try:
```

Comment:
why a try ? I thing the line #29 without the try could be :ok_hand: for anythin in **production**

Reply:
Was there only for a catch if number of processors was not passed. Want me to remove it?

Reply:
In fact I'm not sure handling a failure case of a `core` **python** is our scope.
Yeah, I pref to remove it